index.js

```diff
@@ -2179,6 +2184,13 @@
     };
   }
 
+  //  see :: (String, { name :: String, url :: String }) -> String
+  function see(label, record) {
+    return record.url &&
```

Comment:
```sh
$ npm i sanctuary-type-classes@4
$ npm i sanctuary-def@latest
node
```

```js
var Z = require('sanctuary-type-classes')
var $ = require('sanctuary-def')
var def = $.create({checkTypes: true, env: $.env})
var f = $.UnaryTypeVariable('f')
var a = $.TypeVariable('a')
var b = $.TypeVariable('b')
var map = def('map', {f: [Z.Functor]}, [$.Function([a, b]), f(a), f(b)], Z.map)

map(x => x + 1, 1)
```

```text
TypeError: Type-class constraint violation

map :: Functor f => (a -> b) -> f a -> f b
       ^^^^^^^^^                ^^^
                                 1

1)  1 :: Number

‘map’ requires ‘f’ to satisfy the Functor type-class constraint; the value at position 1 does not.
undefined
```

----

So indeed. It's not a breaking change to use an outdated type-classes library, but it *is* going to lead to a trailing "undefined" at the end of your error messages.